### PR TITLE
Fixing duraton -> duration typo

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

**Comment by dhermes** [comment]
`/approve`

File: content/en/docs/concepts/certificaterequest.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ spec:
   - signing
   - digital signature
   - server auth
-  duraton: 90d
+  duration: 90d
   issuerRef:
     name: ca-issuer
```
Comment (dhermes) [inline content/en/docs/concepts/certificaterequest.md]: While we're at it, the line

> ... attempt to request the `Issuer` `letsencrypt-prod` in the default issuer group `cert-manager.io` ...

seems to be incorrect. Should it not refer to `ca-issuer` (from line 32) as the issuer?

Comment (munnerz) [reply content/en/docs/concepts/certificaterequest.md]: :+1: yep it should - nice spot 😄

**Pull request merged**